microprofile-fault-tolerance/src/test/java/org/jboss/eap/qe/microprofile/fault/tolerance/integration/metrics/MultipleMetricsExtensionTest.java

```diff
@@ -0,0 +1,402 @@
+                .addPackages(true, HelloService.class.getPackage())
+                .addAsManifestResource(ConfigurationUtil.BEANS_XML_FILE_LOCATION, "beans.xml")
+                .addAsManifestResource(new StringAsset(mpConfig), "microprofile-config.properties");
+    }
+
+    /**
+     * Deploy all deployments, so it's possible to get their URLs in other tests. This is limitation of Arquillian
+     * when deployments are deployed manually (in tests) and thus their URL is not known in time when test is started.
+     * The only way how to workaround this issue is to deploy all deployments in first test which allows to inject
+     * deployment URL as params in other test methods.
+     */
+    @Test
+    @InSequence(0)
+    public void deployAll() {
+        deployer.deploy(FAULT_TOLERANCE_DEPLOYMENT);
+        deployer.deploy(FAULT_TOLERANCE_DEPOYMENT_WITH_MP_TELEMETRY);
+        deployer.deploy(FAULT_TOLERANCE_DEPOYMENT_WITH_MP_TELEMETRY_DISABLING_MICROMETER);
+
+        deployer.undeploy(FAULT_TOLERANCE_DEPLOYMENT);
+        deployer.undeploy(FAULT_TOLERANCE_DEPOYMENT_WITH_MP_TELEMETRY);
+        deployer.undeploy(FAULT_TOLERANCE_DEPOYMENT_WITH_MP_TELEMETRY_DISABLING_MICROMETER);
+    }
+
+    /**
+     * Test to verify that no MicroProfile FaultTolerance related metrics are visible via the OTel collector Prometheus
+     * endpoint, when both the Micrometer and MicroProfile Telemetry extensions are not <i>available</i>.
+     *
+     * @param deploymentUrl Base {@link URL} of the app deployment
+     * @throws Exception When something fails during server configuration
+     */
+    @Test
+    @InSequence(10)
+    public void noMetricsAreCollectedWhenMetricsExtensionsAreNotAvailable(
+            @ArquillianResource @OperateOnDeployment(FAULT_TOLERANCE_DEPLOYMENT) URL deploymentUrl) throws Exception {
+        // Remove the MP Telemetry extension
+        MicroProfileTelemetryServerConfiguration.disableMicroProfileTelemetry();
+        // And be sure Micrometer is not available as well
+        MicrometerServerConfiguration.disableMicrometer();
+        // start the OTel collector container
+        OpenTelemetryCollectorContainer otelCollector = OpenTelemetryCollectorContainer.getNewInstance();
+        otelCollector.start();
+        try {
+            // deploy
+            deployer.deploy(FAULT_TOLERANCE_DEPLOYMENT);
+            try {
+                // call the app operation for some times
+                for (int i = 0; i < REQUEST_COUNT; i++) {
+                    get(deploymentUrl + "?operation=timeout&context=foobar&fail=true").then()
+                            .assertThat()
+                            .statusCode(HttpStatus.SC_OK)
+                            .body(containsString("Fallback Hello, context = foobar"));
+                }
+                // give it some time to actually be able and report some metrics via the Pmetheus URL
+                Thread.sleep(5_000);
```

Comment:
Can we decrease this a little bit by adding "otel.metric.export.interval=100" to mp config property file?

Reply:
Thanks for your suggestion, experimenting with it now.